friends/src/App.js

```diff
@@ -1,20 +1,72 @@
 import React, { Component } from "react";
-import { BrowserRouter as Router, Route } from "react-router-dom";
+import { BrowserRouter as Router, Route, NavLink } from "react-router-dom";
 import axios from "axios";
-import FriendForm from "./FriendContainer/FriendForm";
-import FriendSection from "./FriendContainer/FriendSection";
-import Friend from "./FriendContainer/Friend";
+import styled from "styled-components";
+import FriendForm from "./components/FriendForm";
+import FriendSection from "./components/FriendSection";
+import Friend from "./components/Friend";
+
+const StyledFriend = styled.div`
```

Comment:
You might want to put your styling below your component logic to improve code readability